src/oss-tests/ProjectManagerTests/PyPIProjectManagerTests.cs

```diff
@@ -163,7 +176,7 @@ public async Task DetailedPackageExistsAsync_WorksAsExpected(string purlString,
         [DataTestMethod]
         [DataRow("pkg:pypi/pandas@1.4.2", true)]
         [DataRow("pkg:pypi/pandas@12.34.56.78", false)]
-        [DataRow("pkg:pypi/plotly@3.7.1", true)]
+        [DataRow("pkg:pypi/plotly@5.7.0", true)]
```

Comment:
why did you change the version?

Reply:
The http responses in these unit tests rely on a mock HttpMessageHandler, which retrieves pre-saved json files in the /TestData/PyPI folder (rather than actually fetching from server). I changed the version here to use an existing json file rather than adding another to the test data folder.

Reply:
The reason the unit test was passing before is because DetailedPackageExistsAsync() by default calls PackageExistsAsync(), which before this PR used to call the non-version specific endpoint. This would pull the pre-saved json test data for plotly, but once I changed PackageExistsAsync() to poll the specific version, it would fail because we don't have a pre-saved json file for plotly@3.7.1. Since we do already have test data for plotly@5.7.0, I changed it to that package version.